pkgs/applications/blockchains/lighthouse/default.nix

```diff
@@ -84,6 +85,8 @@ rustPlatform.buildRustPackage rec {
   LIGHTHOUSE_DEPOSIT_CONTRACT_SPEC_URL = "file://${depositContractSpec}";
   LIGHTHOUSE_DEPOSIT_CONTRACT_TESTNET_URL = "file://${testnetDepositContractSpec}";
 
+  OPENSSL_NO_VENDOR = true;
```

Comment:
```suggestion
  env.OPENSSL_NO_VENDOR = true;
```

Reply:
Inconsistently with the rest of the package?

Reply:
I'd use it everywhere but feel free to only do so for packages that already use `env` or specify no environment variables.

Reply:
That's the thing I don't understand about `env` — which variables go in it, since virtually everything is an environment variable? buildInputs is an environment variable but doesn't go in env. Is it special just because it's interpreted by Nix code?  So then what about variables that are interpreted by stdenv or hooks at build time? Do they go in env, even though that's basically an implementation detail, especially in the stdenv case?  Or is env just for variables that no Nix code, nor any hooks care about?

Reply:
I think they are different in a structured attributes world, e.g. `env` has to contain valid environment variable contents rather than arbitrary structured data. (They might also get processed in different ways?)

Reply:
env is exported to the environment during the build phase when structuredAttrs are enabled. Other variables are not available for the build tool(s). At least that's how I understand it.

Reply:
Great, thanks for the explanation.